### ISSUE TITLE
Not working with remote interpreter

### ISSUE THREAD

**Issue by icanhazstring** [Haehnchen/idea-php-generics-plugin#39]
Hi, thanks for the plugin!

Unfortunately there seem to be a little bug using it with remote, or at least "project interpreter" (which is a remote one). If I select this, and try to configure the path, the "local" settings just get multiplied.

![image](https://user-images.githubusercontent.com/883543/80948695-ef067000-8df2-11ea-80c9-0efc1e8ea68d.png)

**Comment by siggidiel** [comment]
Same here on Windows. I am using vagrant box (Linux) and selecting the "Remote Interpreter" dropdown item does not trigger any functionality, so that i can pick a interpreter or something. The error message below, that there has no executable been selected, is still there. And yeah, the Local thingy multiplie.

**Comment by j-koehler** [comment]
I tried it using the workaround from https://youtrack.jetbrains.com/issue/WI-36122#focus=streamItem-27-3194637.0-0 with  PhpStorm 2020.1.1 and Plugin version  0.7.0.

I added phpstan/phpstan:0.12 as a docker based CLI interpreter. After doing this, I'm able to pick the newly created interpreter and add the configuration. Based on this [Dockerfile](https://hub.docker.com/r/phpstan/phpstan/dockerfile), I changed volume and mapping to `/app` and set `phpstan` as the "PhpStan path" (validation succeeds - detects 0.12.25 atm).

After this, I enable PhpStan inspections but run into trouble here. Event log says:

```
PHPStan
    phpstan: Can not correctly run the tool with parameters:
    analyse
    --error-format=checkstyle
    /opt/phpstan_temp.tmp/path/to/my/File.php
    Possible tool process hangup after 5 sec.
```
I wonder where this `/opt/phpstan_temp.tmp` comes from? PhpStorm helper?

Sometimes I also get the following error:

```
PHPStan
    No response from phpstan after 5000 ms
```

Maybe it means the container is not running? I wonder how the CLI remote interpreter handling in PhpStorm works aka "when/how/by whom are the containers started".
